src/main/java/com/dnd/gongmuin/credit_history/domain/CreditType.java

```diff
@@ -25,11 +29,10 @@ public static CreditType of(String input) {
 			.orElseThrow(IllegalArgumentException::new);
 	}
 
-	public static CreditType fromDetail(String detail) {
+	public static List<CreditType> fromDetail(String detail) {
 		return Arrays.stream(values())
 			.filter(type -> type.isDetailEqual(detail))
-			.findAny()
-			.orElseThrow(IllegalArgumentException::new);
+			.collect(Collectors.toList());
```

Comment:
```suggestion
			//.collect(Collectors.toList());
			.toList();
```
이렇게 바꾸면 더 간결해질 것 같아요!

Reply:
확 간결해졌네요 :)
둘의 차이가 뭔지 궁금해서 찾아보니깐 `toList()`는 불변 List를 반환하고, `collect(Collectors.toList())`는 가변 List를 반환하네요!